api/service.py

```
@@ -293,6 +293,87 @@ def get_current_username(credentials: HTTPBasicCredentials = Depends(security)):
+def inject_chaos_fault(fault_type: str, duration_seconds: int) -> dict:
+    """Inject a chaos fault for the specified duration."""
+    expiration = time.time() + duration_seconds
+    active_faults[fault_type] = expiration
+    return {
+        "status": "injected",
+        "fault": fault_type,
+        "expires_at": expiration
+    }
+
+
+def create_response(status: str, data: dict = None, **kwargs) -> dict:
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)